### ISSUE TITLE
Show tables scrollable on mobile

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/text#2412]
On small screens it's hard to see the table contents. 
Maybe it's possible the make the table scrollable?

**Comment by teneighty-yvr** [comment]
First, thank you for adding table support! This is a fantastic feature for my organization!

I propose adding horizontal scrolling for all screen sizes. I was already using tables with the "plain text editor" application, but the first one I opened with this application after upgrading to NC 24 had a wide left margin, because the table is left aligned with the text area which is centered and 648 wide even though my display is 1920x1080, and the right edge was well past the right edge of the screen. 

I also noticed that if I tab my way across the columns, it will show the last columns, but when I wrap around to the first column again, it does not return the table to it's original position. I actually can't see the first column at this point. Maybe that is a Firefox bug?

An important consideration for any fix is to utilize the entire available screen width effectively. I don't know how complicated that would be, given the document is contained in the narrower than my display div.ProseMirror. My HTML/CSS is pretty limited, but I think it's clear what I'm proposing?

Horizontal scrolling the whole document could also benefit plain text where, at the moment, lines are not wrapped so long lines have the same problem as tables.

my browser: Firefox 101.0
